code/modules/food_and_drinks/kitchen_machinery/monkeyrecycler.dm

```diff
@@ -14,15 +16,24 @@
 	var/cube_production = 1
 	var/cycle_through = 0
 	var/obj/item/reagent_containers/food/snacks/monkeycube/cube_type = /obj/item/reagent_containers/food/snacks/monkeycube
+	var/list/connected = list()
 
 /obj/machinery/monkey_recycler/New()
 	..()
 	component_parts = list()
 	component_parts += new /obj/item/circuitboard/monkey_recycler(null)
 	component_parts += new /obj/item/stock_parts/manipulator(null)
 	component_parts += new /obj/item/stock_parts/matter_bin(null)
+	GLOB.monkey_recyclers += src
 	RefreshParts()
 
+/obj/machinery/monkey_recycler/Destroy()
+	if(src in GLOB.monkey_recyclers)
```

Comment:
This line isn't necessary; if you try to remove something from a list and it's not in that list, it doesn't create problems, it just returns.

Reply:
sorry I have a habit of pasting over the TG code and not changing it